test/unit/ui/map_events.test.js

```diff
@@ -553,3 +553,35 @@ test(`Map#on mousedown doesn't fire subsequent click event if mousepos changes`,
     map.remove();
     t.end();
 });
+
+test(`Map#on mousedown fires subsequent click event if mousepos changes less than drag threshold`, (t) => {
```

Comment:
Minor again, but I'd spell out `mouse position` instead of `mousePos` in these two test descriptions.